src/features/unified-share-modal/__tests__/UnifiedShareModal.test.js

```diff
@@ -53,6 +53,15 @@ describe('features/unified-share-modal/UnifiedShareModal', () => {
             expect(wrapper).toMatchSnapshot();
         });
 
+        test.only('should render a allShareRestrictionWarning message when it is available', () => {
```

Comment:
Minor: "should render *an*"

Reply:
and should not have the .only, will remove